### PR TITLE
Adding TravisCI to build the app in every commit or PR

### PR DESCRIPTION
**Pull request by sbritocorral**
Simple Travis CI integration for android project . Travis will make a build on every commit or Pull Request.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+branches:
+  only:
+  # Build only for master PR or Commit
+  - master
+
+env:
+ global:
+    # Global variables to get things easier to change
+    - ANDROID_API_LEVEL_TARGET=28
+    - ANDROID_API_LEVEL_EMULATOR=22
```
Comment (xiamaz) [inline .travis.yml]: Your API levels are too high. Our minimum requirement is android 5.0 (API level 21)

Comment (irux) [reply .travis.yml]: @xiamaz do you mean because the emulator API ?

Comment (alejo096) [reply .travis.yml]: @xiamaz do you mean because of the Emulator API ??

Comment (sbritocorral) [reply .travis.yml]: if you check the build.gradle file you see that compiledSdkVersion is 28,  thats why you install API 28 and it works for 21-28. The minimum its also specified there

**Review by lbb — APPROVED** [review]
LGTM

**Pull request merged**